### PR TITLE
has systemd double check

### PR DESCRIPTION
**Pull request by k-9527**
What type of PR is this?
/kind feature

What this PR does / why we need it:
In my case, running `keadm int` in a simplified centos machine.there is no `file` command existed,so hasSystemd() always return false,but systemd is surely running there.After false return,we can do check it again by sb_booted.(reference http://www.freedesktop.org/software/systemd/man/sd_booted.html)

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

### CODE REVIEW

**Comment by daixiang0** [comment]
lgtm

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 1]
```diff
@@ -420,14 +422,20 @@ func isEdgeCoreServiceRunning(serviceName string) (bool, error) {
 }
 
 // check if systemd exist
+// if command run failed,then check it by sd_booted
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/util/common.go:425]:
```suggestion
// if command run failed, then check it by sd_booted
```

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 2]
```diff
@@ -420,14 +422,20 @@ func isEdgeCoreServiceRunning(serviceName string) (bool, error) {
...
+// reference http://www.freedesktop.org/software/systemd/man/sd_booted.html
 func hasSystemd() bool {
 	cmd := "file /sbin/init"
 
-	if err := NewCommand(cmd).Exec(); err != nil {
+	if err := NewCommand(cmd).Exec(); err == nil {
+		return true
+	}
+	//checks whether `SystemdBootPath` exists and is a directory
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/util/common.go:433]:
```suggestion
	// checks whether `SystemdBootPath` exists and is a directory
	// reference http://www.freedesktop.org/software/systemd/man/sd_booted.html
```

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 3]
```diff
@@ -420,14 +422,20 @@ func isEdgeCoreServiceRunning(serviceName string) (bool, error) {
 }
 
 // check if systemd exist
+// if command run failed,then check it by sd_booted
+// reference http://www.freedesktop.org/software/systemd/man/sd_booted.html
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/util/common.go:426]:
```suggestion
```

**Comment by fisherxu** [comment]
@k-9527 Please sign off to fix the DCO ci issue :)

**Comment by k-9527** [comment]
> @k-9527 Please sign off to fix the DCO ci issue :)
copy that

File: keadm/cmd/keadm/app/cmd/util/common.go (hunk 2 above)
Comment (k-9527) [reply keadm/cmd/keadm/app/cmd/util/common.go]: ok

File: keadm/cmd/keadm/app/cmd/util/common.go (hunk 3 above)
Comment (k-9527) [reply keadm/cmd/keadm/app/cmd/util/common.go]: deleted

**Comment by fisherxu** [comment]
@k-9527 please fix the failed DCO ci, thanks.

**Comment by k-9527** [comment]
fixed

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**